ci/utils.sc

```diff
@@ -136,3 +136,20 @@ def escapeCmdArg(cmd: String): String = {
   val subbed = cmd.replace("'", "\\'").replace("\n", "\\n")
   s"""$$'${subbed}'"""
 }
+
+case class SemVer(major: Int, minor: Int, build: Int, commit: String) {
+  def toStringtWithCommit(): String = s"$major.$minor.$build-$commit"
```

Comment:
No prefix here? Perhaps this should just be the default `toString`?

Reply:
Good point.